.github/workflows/docs.yml

```diff
@@ -0,0 +1,27 @@
+name: Documentation
+
+on:
+  push:
+    branches: ["main"]
+    paths:
+      - "@typespec/**"
+
+jobs:
+  push-JSON-schema:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Push directory to another repository
+        uses: cpina/github-action-push-to-another-repository@v1.7.2
+        env:
+          SSH_DEPLOY_KEY: ${{ secrets.SSH_DEPLOY_KEY }}
+        with:
+          # Source directory from the origin directory
+          source-directory: "@typespec/json-schema"
+          # Name of the destination username/organization
+          destination-github-username: "keenawa-co"
+          # Destination repository
+          destination-repository-name: "moss-studio-docs"
+          # Email for the git commit
+          user-email: github-actions[bot]@users.noreply.github.com
```

Comment:
The field names are quite descriptive, so there’s no need to clutter the file with redundant comments